configurations/rolling-upgrade-with-sla.yaml

```diff
@@ -0,0 +1,14 @@
+# workloads
+stress_before_upgrade: "cassandra-stress write cl=QUORUM n=60000000 -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native -rate threads=1000 -pop seq=1..60000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5"
+stress_during_entire_upgrade: [ "cassandra-stress read cl=QUORUM duration=30m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -mode cql3 native <sla credentials 0> -rate threads=200 -pop seq=1..30000000 -col 'n=FIXED(10) size=FIXED(512)' -log interval=5",
```

Comment:
did you took a look at the monitoring, that this 30min is enough for the whole cluster upgrade ?

Reply:
Because of failure I can't to check it. The test failed before

Reply:
o.k. good enough for now